dev_tools.sh

```
@@ -7,11 +7,62 @@ format_code() {
 }
 
 generate_proto() {
-    cd src/
-    protoc proto/action_space.proto --python_out=craftground --pyi_out=craftground
-    protoc proto/initial_environment.proto --python_out=craftground --pyi_out=craftground
-    protoc proto/observation_space.proto --python_out=craftground --pyi_out=craftground
-    protoc proto/action_space.proto --java_out=../shared-java/src/main/java/ --kotlin_out=../shared-java/src/main/java/
-    protoc proto/initial_environment.proto --java_out=../shared-java/src/main/java/ --kotlin_out=../shared-java/src/main/java/
-    protoc proto/observation_space.proto --java_out=../shared-java/src/main/java/ --kotlin_out=../shared-java/src/main/java/
+    (
+        cd src/
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)